.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: check-docstring-first
+      - id: check-toml
+      - id: check-yaml
+        exclude: packaging/.*
+      - id: mixed-line-ending
+        args: [--fix=lf]
+      - id: end-of-file-fixer
+
+
+  - repo: https://github.com/omnilib/ufmt
+    rev: v1.3.0
+    hooks:
+      - id: ufmt
+        additional_dependencies:
+          - black == 21.9b0
+          - usort == 0.6.4
+
+
+  -   repo: https://github.com/pre-commit/mirrors-clang-format
```

Comment:
Nit / Question / Suggestion

I'm not familiar with torchtext CI job details, but this is possibly redundant with the `run_style_checks.sh` job? You could also have a `flake8` hook done here (like in torchvision) and possibly remove `run_style_checks.sh` entirely.